plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -453,10 +453,6 @@ private String getCommandLogTitle(Command cmd) {
 
     @Override
     public Answer executeRequest(Command cmd) {
-
-        if (s_logger.isTraceEnabled())
-            s_logger.trace("Begin executeRequest(), cmd: " + cmd.getClass().getSimpleName());
-
```

Comment:
@SadiJr Better to call _logCommand()_ here (or somewhere at L#466 before executing the actual cmd method), instead adding it in all the cmd execute methods.

Reply:
Done, thanks.